transformer_engine/jax/flax/transformer.py

```diff
@@ -483,6 +494,8 @@ class DotProductAttention(nn.Module):  # pylint: disable=too-few-public-methods
     scale_factor: Optional[float] = None
     transpose_batch_sequence: bool = True
     window_size: Optional[Tuple[int, int]] = None
+    context_parallel_causal_load_balanced: bool = False
+    context_parallel_axis: str = ""
```

Comment:
We need to raise a `ValueError` when users want to use context parallel but there is no fused_attn backend (fall back to _UnfusedDotProductAttention), since we don't have context parallel support for it.